<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgManageParaSetting</class>
 <widget class="QDialog" name="DlgManageParaSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>941</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout" stretch="1,1">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupEddy">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>电涡流参数</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>901</width>
         <height>241</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="LayoutEddy">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <row>
           <property name="text">
            <string>电涡流</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>角度</string>
           </property>
          </row>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="group7200">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>7200激光数据</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>911</width>
         <height>251</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="Layout7200">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupLaser7200A">
          <property name="title">
           <string>A轨</string>
          </property>
          <widget class="QWidget" name="gridLayoutWidget">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>30</y>
             <width>241</width>
             <height>178</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0" columnstretch="3,1,1,3" columnminimumwidth="1,1,0,0">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <property name="horizontalSpacing">
             <number>7</number>
            </property>
            <property name="verticalSpacing">
             <number>20</number>
            </property>
            <item row="3" column="1">
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>旋转角度:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>安装距离:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>安装高度:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>倾斜角度:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="lineEdit_2"/>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="lineEdit_3"/>
            </item>
            <item row="3" column="2">
             <widget class="QLineEdit" name="lineEdit_4"/>
            </item>
            <item row="1" column="3">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupLaser7200B">
          <property name="title">
           <string>B轨</string>
          </property>
          <widget class="QWidget" name="gridLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>30</y>
             <width>241</width>
             <height>178</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_2" columnstretch="3,1,1,3" columnminimumwidth="1,1,0,0">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <property name="horizontalSpacing">
             <number>7</number>
            </property>
            <property name="verticalSpacing">
             <number>20</number>
            </property>
            <item row="1" column="1">
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>安装距离:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>安装高度:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>旋转角度:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLineEdit" name="lineEdit_8"/>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="lineEdit_5"/>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label_8">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>倾斜角度:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="lineEdit_6"/>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="lineEdit_7"/>
            </item>
            <item row="1" column="0">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="3">
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
